components/MovieDetail/index.js

```diff
@@ -189,6 +222,20 @@ export default function MovieDetail({ movie }) {
           </StyledButton>
         </StyledSectionButtons>
       </section>
+      <StyledShowTrailerButton onClick={displayTrailer}>
+        {showTrailer ? "Hide the trailer" : "Watch the trailer"}
+      </StyledShowTrailerButton>
+      <section>
+        {showTrailer ? (
```

Comment:
Not sure but maybe `{showTrailer && <ReactPlayer...}` is working too and you could save some lines of code 🙂

Reply:
I will look into it. Thanks!